ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-ref.yaml

```diff
@@ -30,7 +30,7 @@ ref:
     documentation: |-
       The number of iterations per worker nodes to create
   commands: openshift-qe-cluster-density-v2-commands.sh
-  timeout: 3h
+  timeout: 6h
```

Comment:
@krishvoor should we just set this to max 8h to be safe since cluster is very large?